packages/components/src/ui/control-group/types.ts

```diff
@@ -12,7 +12,7 @@ import type { FlexProps } from '../../flex/types';
 export type ControlGroupContext = {
 	isFirst?: boolean;
 	isLast?: boolean;
-	isMidde?: boolean;
+	isMiddle?: boolean;
```

Comment:
😅 I'm surprised this didn't cause an error before?

Reply:
It looks like the repo doesn't contain any usage of the `<ControlGroup />` component, and that there are no specific tests for the `ControlGroupContext`.

Maybe we should add a test for that in a separate PR?